README.md

```diff
@@ -32,6 +31,12 @@ docker run -p 8080:8080 simplifi/docker-presto:latest
 
 Configuration is handled by setting the environment variables outlined below.
 
+### jvm.config
+
+| Property         | Environment Variable    | Default Value                 | Description                                                    |
+| -----------------| ----------------------- | ----------------------------- | -------------------------------------------------------------- |
+| Xmx              | PRESTO_JVM_XMX_GB       | 16                            | the maximum size, in gigabytes, of the memory allocation pool. |
```

Comment:
I wonder if it would be better to just allow passing a whole long string of JVM_ARGUMENTS or something like that?

Reply:
Thats a good point, my current approach could get out of hand pretty quick.  I'll give that a shot.

Reply:
I'm kind of torn on how to approach this, I'm going to back out my JVM config changes from this PR and I'll address it in it's own PR once I have had a chance to give it some more thought.